src/js/components/Marathon.jsx

```diff
@@ -252,6 +254,46 @@ var Marathon = React.createClass({
         wait: true
       });
     }
+    /*eslint-enable no-alert */
+  },
+
+  destroyDeployment: function (deployment, options, component) {
```

Comment:
That's probably a merge/rebase-error. This function isn't there anymore due to a refactoring.